src/main/scala/cromwell/engine/callexecution/BackendCallExecutionActor.scala

```diff
@@ -0,0 +1,27 @@
+package cromwell.engine.callexecution
+
+import akka.event.{Logging, LoggingAdapter}
+import cromwell.engine.backend._
+import cromwell.logging.WorkflowLogger
+
+import scala.language.postfixOps
+import CallExecutionActor._
+
+/**
+  * Actor to manage the execution of a single backend call.
+  * */
+class BackendCallExecutionActor(backendCall: BackendCall) extends CallExecutionActor {
+  override val logger = WorkflowLogger(
+    this.getClass.getSimpleName,
+    backendCall.workflowDescriptor,
+    akkaLogger = Option(akkaLogger),
+    callTag = Option(backendCall.key.tag)
+  )
```

Comment:
I can't tell whether you wanted another newline below this (which I added) or wanted this to NOT be on a newline.